recipes/libelfin/all/patches/946dde57e5caef8297b9339f3a7971401d540840.patch

```diff
@@ -0,0 +1,165 @@
+diff --git a/dwarf/abbrev.cc b/dwarf/abbrev.cc
+index c780b8e..f77dc0c 100644
+--- a/dwarf/abbrev.cc
++++ b/dwarf/abbrev.cc
+@@ -133,6 +133,10 @@ resolve_type(DW_AT name, DW_FORM form)
+                 case DW_AT::ranges:
+                         return value::type::rangelist;
+ 
++                case DW_AT::lo_user...DW_AT::hi_user:
++                        //HACK: ignore vendor extensions
++                        return value::type::invalid;
++
+                 default:
+                         throw format_error("DW_FORM_sec_offset not expected for attribute " +
+                                            to_string(name));
+diff --git a/dwarf/dwarf.cc b/dwarf/dwarf.cc
+index fe9b947..2465eef 100644
+--- a/dwarf/dwarf.cc
++++ b/dwarf/dwarf.cc
+@@ -289,8 +289,7 @@ compilation_unit::get_line_table() const
+ {
+         if (!m->lt.valid()) {
+                 const die &d = root();
+-                if (!d.has(DW_AT::stmt_list) || !d.has(DW_AT::name) ||
+-                    !d.has(DW_AT::comp_dir))
++                if (!d.has(DW_AT::stmt_list) || !d.has(DW_AT::name))
+                         goto done;
+ 
+                 shared_ptr<section> sec;
+@@ -300,8 +299,10 @@ compilation_unit::get_line_table() const
+                         goto done;
+                 }
+ 
++                auto comp_dir = d.has(DW_AT::comp_dir) ? at_comp_dir(d) : "";
++                
+                 m->lt = line_table(sec, d[DW_AT::stmt_list].as_sec_offset(),
+-                                   m->subsec->addr_size, at_comp_dir(d),
++                                   m->subsec->addr_size, comp_dir,
+                                    at_name(d));
+         }
+ done:
+diff --git a/elf/elf++.hh b/elf/elf++.hh
+index 562d1de..ee59ed0 100644
+--- a/elf/elf++.hh
++++ b/elf/elf++.hh
+@@ -48,7 +48,7 @@ public:
+  * section data.  Hence, callers must ensure that the loader passed to
+  * this file remains live as long as any such pointer is in use.
+  * Keeping any object that can return such a pointer live is
+- * sufficieint to keep the loader live.
++ * sufficient to keep the loader live.
```

Comment:
There's a few changes to the comments in the patch, I am not sure that's the place of C3i

Reply:
Related to the question about the patch above.